src/test/java/uk/nhs/prm/deduction/e2e/utility/TestUtils.java

```diff
@@ -161,4 +162,12 @@ public static LargeEhrTestFiles prepareTestFilesForLargeEhr(
 
         return new LargeEhrTestFiles(largeEhrCore, largeEhrFragment1, largeEhrFragment2, ehrRequest, continueRequest);
     }
+
+    public static String createUuidAsString() {
```

Comment:
```suggestion
    public static String createUUIDAsString() {
```

Could we capitalise UUID, please?